### DISCUSSION TITLE
Future Ready Talent

### DISCUSSION THREAD

**Discussion by Chetan50** [github/india#542] [Students]
I have completed 2 modules but then the status is in "IN PROGRESS". Can anyone of you help me to overcome this challenge?

**Comment by Vish-devop** [comment]
Hello, I am writing the email to future ready talent regarding the github student developer pack .
I have already completed all the steps that future ready talent is asking .
But, haven't got my developer pack, yet.
Could any-one help me in resolving my issue.

**Comment by Vish-devop** [comment]
> I have completed 2 modules but then the status is in "IN PROGRESS". Can anyone of you help me to overcome this challenge?

This same problem is happening with me,
I have completed 2 industry sessions but still on the website the toggle button is showing - incomplete.
If anyone could tell me the issue it would be very helpfull.
@FutureReadyTalent
